test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -727,6 +727,40 @@ def forward(self, a, b, c, d):
         d = torch.randn(2, 3)
         self.run_test(MyModel(), (a, b, c, d))
 
+
+    def test_loop_output(self):
+        class PaddingLayer(torch.jit.ScriptModule):
+            def __init__(self):
+                super(PaddingLayer, self).__init__()
+
+            @torch.jit.script_method
+            def forward(self, input_t, shape_t):
+                # type: (Tensor, Tensor) -> Tensor
+                output_size = [len(shape_t), int(shape_t.max())]
+                model_input = torch.zeros(size=output_size, dtype=input_t.dtype, device=input_t.device)
+                s_start = 0
+                for s_id, s_len in enumerate(shape_t):
```

Comment:
test case depends on prim_min, CI failing

Reply:
prim_min PR is merged